sdk/scheduler/src/main/java/com/mesosphere/sdk/curator/CuratorPersister.java

```diff
@@ -313,10 +313,9 @@ public void recursiveDelete(String unprefixedPath) throws PersisterException {
       LOGGER.debug("Deleting children of root {}", path);
       try {
         client
-            .inTransaction()
+            .transactionOp()
```

Comment:
need to add this to the list of operations below so its comitted

Reply:
good catch, turns out this is already done in the next lines, deleted the code.